simulation.py

```diff
@@ -0,0 +1,73 @@
+"""
+A function to simulate alpha
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import TRADING_CONFIG, DATA_LIST_ALL
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = raw_data['workdays']
+    invest_data = {index : value
+                   for index, value in raw_data.items() if index != 'workdays'}
+    return_data = {}
+    for key, value in workdays.items():
+        return_data[key] = copy.deepcopy(invest_data)
```

Comment:
Not really sure what you want here. Can you explain what we are doing here, and why we need `deepcopy` to achieve it?

Reply:
Cause _return_data[region]_ means the workdays data in that region.
Each region have different workday data, it we don't _deepcopy_ the pure data, that will be polluted while set_workdays in line 26.

Reply:
While simulation is an one time process now, it may not need this operation.

Reply:
I see. I think it's fine to keep the current version if we will update it to multi-process version soon.